packages/tanstack-solid-query/src/internals/subscriptionOptions.ts

```
@@ -0,0 +1,309 @@
+          reset,
+        }
+      : {
+          data: undefined,
+          error: null,
+          status: 'idle',
+          reset,
+        };
+  };
+
+  createEffect(() => {
+    reset;
```

ESLint

[error] Expected an assignment or function call and instead saw an expression.

(@typescript-eslint/no-unused-expressions)